### ISSUE TITLE
Unable to approve/appoint applicants.

### ISSUE THREAD

**Issue by jkmy23** [NotEnoughTina/FirstLady#1]
Hi there, thank you so much for writing this up. Its is excellent. However, I seem to have succumbed to a problem with the bot appointing new applicants. 
It pretty much goes into each tile and finds the applicants, but somehow it won't click on the green tick to approve it. 
I have looked at the code but unfortunately I am not much of a developer, hopefully you're able to tell me what needs to be tweaked. 
Otherwise, I can see it looping on all the tiles and selecting the right areas.
If it helps, here are the debug logs:
(venv) c:\FirstLadyBot>py main.py
Connected devices: ['emulator-5554']
Using device: emulator-5554
[2024-10-23 21:51:27] Checking strategy
No matches found.
        Accepted 1 people for strategy.
[2024-10-23 21:52:00] Checking security
No matches found.
        Accepted 1 people for security.
[2024-10-23 21:52:33] Checking development
No matches found.
        Accepted 1 people for development.
[2024-10-23 21:53:09] Checking science
No matches found.
        Accepted 1 people for science.
[2024-10-23 21:53:42] Checking interior
No matches found.
        Accepted 1 people for interior.
[2024-10-23 21:54:19] Checking strategy
No matches found.
        Accepted 1 people for strategy.

Thanks and love your work.

**Comment by NotEnoughTina** [comment]
Replicated the issue on another device. Updated the image detection logic to return the center of the icon to ensure accurate clicking, preventing missed clicks.

Please pull the latest updates and give it a try. Let me know if this resolves the issue on your end.

**Comment by jkmy23** [comment]
Hi NotEnoughTina,

It definitely works better now, the next problem is that it starts taking a screenshot from the "best match" app and it doesn't know how to close the window down. Until I close down the screenshot manually, it won't continue. 

But I will pull your latest code and try again. The new image detection threshold definitely did the trick.

**Comment by jkmy23** [comment]
The new code doesn't work anymore. The new scale range may have broken it.

**Comment by NotEnoughTina** [comment]
Glad to hear it's working better for you!

Regarding the screenshot popping up, that happens when the debug flag is set to `True` [here](https://github.com/NotEnoughTina/FirstLady/blob/f8a24688e34ac70aaea0778b1294e299aff2903b/main.py#L52). You can prevent it by setting the debug flag to `False`.

For the threshold, I moved it to the [config.ini](https://github.com/NotEnoughTina/FirstLady/blob/f8a24688e34ac70aaea0778b1294e299aff2903b/config.ini#L34) for easier adjustment without modifying the code directly.

Regarding the new scale range, were you encountering any specific errors or issues? I'd be happy to help troubleshoot if it's causing problems.

Let me know how it goes after pulling the latest code!

**Comment by jkmy23** [comment]
I've used the first version of your code but modified the threshold to 0.62. I've got this bot running the last 24 hours with no issues. I couldn't get the current code version to work and I didn't spend any time debugging it.
The bot is very efficient, and still works pretty well. I've dropped by BlueStacks performance down to 2 cores and 2GB using only 20FPS. 
Everything is working swimmingly well :) Thanks for the bot, I really like it.
